src/i18n/locale/nl.json

```diff
@@ -566,6 +566,7 @@
   "components.Settings.SettingsJobsCache.radarr-scan": "Radarr-scan",
   "components.Settings.SettingsJobsCache.plex-recently-added-scan": "Plex recent toegevoegde scan",
   "components.Settings.SettingsJobsCache.plex-full-scan": "Plex volledige bibliotheekscan",
+  "components.Settings.SettingsJobsCache.availability-sync": "Volledige beschikbaarheidsscan",
```

Comment:
Fair enough, thought I'd update it just as well because I speak the language, but in that case I'll let the translation team take care of it and remove it from here.

Reply:
You can update it in Weblate yourself once this PR is merged! 💪🏻